Keyword RECURSIVE is missing in ecbuild_bundle for soca

The CMakeLists.txt file  generated from ` jedi_bundle_update_hash  $mydate`  contains a line for soca, showing:
`ecbuild_bundle( PROJECT soca  GIT "https://github.com/JCSDA-internal/soca"  BRANCH 9a5bcee49ceffabac1123cc19c00385304aebbe0 )` 
This line misses `BRANCH xyz  RECURSIVE`.   The ecbuild/cmake starting from this  CMakeLists.txt gives an error:

```
CMake Error at soca/external/mom6/CMakeLists.txt:31 (add_library):
  Cannot find source file:

    MOM6/src/ALE/remapping_attic.F90

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm
  .ccm .cxxm .c++m .h .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90
  .f95 .f03 .hip .ispc

CMake Error at soca/external/mom6/CMakeLists.txt:31 (add_library):
  No SOURCES given to target: mom6

CMake Generate step failed.  Build files cannot be regenerated correctly.
```

There is no build error otherwise. Please add this keyword in the repo.

--- a/src/jedi_bundle/config/bundles/build-order.yaml
+++ b/src/jedi_bundle/config/bundles/build-order.yaml
@@ -52,6 +52,7 @@
 # Soca
 - soca:
     default_branch: develop
+    recursive: true
 
 # Fv3-jedi
 - fv3-jedi-lm:

